apps/src/aichat/views/modelCustomization/SetupCustomization.tsx

```diff
@@ -34,12 +34,13 @@ const SetupCustomization: React.FunctionComponent = () => {
   );
 
   /** defaults to all models if not set in levelProperties */
-  const availableModelIds = useAppSelector(
-    state =>
-      (state.lab.levelProperties as AichatLevelProperties | undefined)
-        ?.aichatSettings?.availableModelIds
-  );
-  const availableModels = availableModelIds
+  const availableModelIds =
+    useAppSelector(
+      state =>
+        (state.lab.levelProperties as AichatLevelProperties | undefined)
+          ?.aichatSettings?.availableModelIds
+    ) || [];
+  const availableModels = availableModelIds.length
```

Comment:
I was getting page load errors here after updating some levelbuilder config -- I think this was returning true b/c `availableModels` was a blank array (truthy), causing an error on line 48 below.

Reply:
cc @cnbrenci, LMK if this sounds right!

Reply:
Yees, this should also be fixed with 

https://github.com/code-dot-org/code-dot-org/blob/65860261b1e055aa65471fec69990ddd21731d8b/apps/src/aichat/views/modelCustomization/constants.ts#L68

so it won't default to empty.

Sorry about that!